packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -39,10 +39,10 @@ export class Copy {
      */
     public static async dataSet(
         session: AbstractSession,
-        { dataSetName: toDataSetName, memberName: toMemberName }: IDataSet,
+        { dsn: toDataSetName, member: toMemberName }: IDataSet,
         options: ICopyDatasetOptions
     ): Promise<IZosFilesResponse> {
-        ImperativeExpect.toBeDefinedAndNonBlank(options.fromDataSet.dataSetName, "fromDataSetName");
+        ImperativeExpect.toBeDefinedAndNonBlank(options["from-dataset"].dsn, "fromDataSetName");
```

Comment:
This seems related to the previous comment, and seems out of place compared to what it used to be.

Reply:
it is out of place 😞 ... im just the lowly merger here though 😄

Reply:
same as above